itests/kit/ensemble_opts.go

```diff
@@ -19,10 +21,16 @@ type ensembleOpts struct {
 	verifiedRoot genesisAccount
 	accounts     []genesisAccount
 	mockProofs   bool
+
+	upgradeSchedule stmgr.UpgradeSchedule
 }
 
 var DefaultEnsembleOpts = ensembleOpts{
 	pastOffset: 10000000 * time.Second, // time sufficiently in the past to trigger catch-up mining.
+	upgradeSchedule: stmgr.UpgradeSchedule{{
```

Comment:
nit: the fact that this is so nicely expressed as `GenesisNetworkVersion(build.NewestNetworkVersion)` makes me wonder if we'd be slightly better off with a DefaultEnsembleOpts of type `[]EnsembleOpts` applied to a zero valued `ensembleOpts`.  The name `ensembleOpts` could be changed to something different, `ensembleCfg`?, to make this a little less confusing.

Not very important just riffing.

Reply:
I agree. I'd much rather have a set of default options (also avoids cases where the default "config" gets modified accidentally). But that'll be a future cleanup.